[REFACTOR] Reworks maintenance fake insulated gloves

Thunder12345 opened the pull request:
Fake yellows from maintenance are now an actual item, instead of being varedited white gloves. Now they can actually be used by admins, mappers, and possibly other loot spawn lists.

Commit:
Reworks fake yellows

Replaces the varedited white gloves in the maint loot spawner list with
a subtype of insulated gloves, with the siemens coefficient changed to 1
and a different desc.

Commit:
Merge remote-tracking branch 'upstream/master' into glovening

Deantwo commented on code/modules/clothing/gloves/color.dm:
```diff
@@ -7,6 +7,10 @@
 	item_color="yellow"
 
+/obj/item/clothing/gloves/color/yellow/fake
+	desc = "These don't feel like rubber..."
```
You weren't gonna make it ```"These gloves will protect the wearer from electric shock. These don't feel like rubber..."```? Seem better as it also explain what they *should* do.

Deantwo commented on code/modules/clothing/gloves/color.dm:
```diff
@@ -7,6 +7,10 @@
+	desc = "These don't feel like rubber..."
+	siemens_coefficient = 1
+
 /obj/item/clothing/gloves/color/fyellow                             //Cheap Chinese Crap
```
Rename ```fyellow``` to ```budget``` and make it a child of ```/obj/item/clothing/gloves/color/yellow``` as discussed on IRC.

Deantwo commented on code/modules/clothing/gloves/color.dm:
```diff
@@ -7,6 +7,10 @@
+	siemens_coefficient = 1
+
 /obj/item/clothing/gloves/color/fyellow                             //Cheap Chinese Crap
 	desc = "These gloves are cheap knockoffs of the coveted ones - no way this can end badly."
```
Just to make it all consistent, why not make it ```"These gloves will protect the wearer from electric shock. These gloves are cheap knockoffs of the coveted ones - no way this can end badly."``` or is that too long?

Commit:
Description change

Added the regular insulated glove description to the start of the fake
glove description.

Deantwo commented:
:+1:
Someone that know how to work with *map merger* can do https://github.com/tgstation/-tg-station/pull/7765#discussion_r24748649 I guess.
